src/sidebar/components/menu-keyboard-navigation.js

```diff
@@ -42,7 +57,8 @@ export default function MenuKeyboardNavigation({
 
   const onKeyDown = event => {
     const menuItems = Array.from(
-      menuRef.current.querySelectorAll('[role^="menuitem"]')
+      /** @type {NodeListOf<HTMLElement>} */
```

Comment:
I was seeing an error down below ...
`src/sidebar/components/menu-keyboard-navigation.js:105:33 - error TS2339: Property 'focus' does not exist on type 'Element'.`

I could either cast here, or there, but I believe that  querySelectorAll returns an Element array, but it needs to be an HTMLElement (or similar).